src/uu/tr/src/operation.rs

```diff
@@ -627,33 +641,32 @@ impl TranslateOperation {
 
 impl SymbolTranslator for TranslateOperation {
     fn translate(&mut self, current: u8) -> Option<u8> {
-        Some(
-            self.translation_map
-                .get(&current)
-                .copied()
-                .unwrap_or(current),
-        )
+        Some(self.translation_table[current as usize])
     }
 }
 
 #[derive(Debug, Clone)]
 pub struct SqueezeOperation {
-    set1: HashSet<u8>,
+    squeeze_table: [bool; 256],
     previous: Option<u8>,
 }
 
 impl SqueezeOperation {
     pub fn new(set1: Vec<u8>) -> Self {
+        let mut squeeze_table = [false; 256];
+        for byte in set1 {
+            squeeze_table[byte as usize] = true;
+        }
```

Comment:
Also duplicated code with the DeleteOperation.

Not sure if a `set_to_bitmap` function is worth it...